synapseclient/models/file.py

```diff
@@ -697,7 +705,7 @@ async def get(
         self.fill_from_dict(synapse_file=entity, set_annotations=True)
 
         if include_activity:
-            self.activity = await Activity.from_parent(
+            self.activity = await Activity.from_parent_async(
```

Comment:
I'm confused as to why the synchronous `from_parent` isn't being used here

Reply:
When we are already in an async context we don't need to run the synchronous version of the code. Only the entry point into the code should ever be calling the non-async version (What the person using the client is calling).